scripts/xcode_build.rb

```diff
@@ -40,6 +43,7 @@
     config.build_settings['LIBRARY_SEARCH_PATHS']   = libraries_search_path
     config.build_settings['OTHER_LINKER_FLAGS']     = other_linker_flags
     config.build_settings['ENABLE_BITCODE']         = 'No'
+    config.build_settings['DEVELOPMENT_TEAM']       = 'GW8XWHWQR7'
```

Comment:
Can we pass in this from command line instead of hardcode?

Reply:
will put it in the org-context in Circle CI